app/k9mail/src/main/AndroidManifest.xml

```diff
@@ -314,7 +314,12 @@
 
         <activity
             android:name=".ui.settings.AboutActivity"
-            android:label="@string/about_action" />
+            android:label="@string/about_action">
+            <intent-filter>
+                <action android:name="com.fsck.k9.OpenAbout" />
```

Comment:
:thinking: I think action names follow the same naming scheme as categories that is last part should be `UPPER_CASE`. E.g. `android.intent.action.MAIN`

Reply:
I saw that K9-specific ones tended to use camel case for actions (though it's a bit of a mixture), eg 
https://github.com/k9mail/k-9/blob/08172c42e1d569ac139f0f20667765756233e0ae/app/k9mail/src/main/AndroidManifest.xml#L346-L356
Is upper cased preferred?